.ado/continuous.yml

```diff
@@ -8,27 +8,6 @@ variables:
   - group: platform-override-zero-permission-token
 
 stages:
-  - stage: Setup
-    jobs:
-      - job: Setup
-        variables:
-          - template: variables/windows.yml
-        pool:
-          vmImage: windows-2019
-        steps:
-          - template: templates/configure-git.yml
-
-          - template: templates/yarn-install.yml
```

Comment:
Is yarn install no longer necessary in CI in Setup?

Reply:
No. This is taken from the PR setup where I did some time based optimizations. A big one was to only install beachball instead of the whole repo.